### ISSUE TITLE
[A11Y] Android: Hidden/Invisible AMPstart link is present on homepage

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#386]
**ENVIRONMENT:**

OS (number): N (7.0)
Build number: NRD91R
Form factor (Smartphone / tablet):  Nexus 9
Browser: Chrome Dev 59.0.3062.4

**Pre-requisite:**
1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)
2. Linearly Navigate to sidebar menu.
3. Try to Linearly Navigate forward.

**ACTUAL RESULTS:**

Focus is lost and talkback says, “Ampstart, link” “double tap to activate”. This link is not visible. If user double taps, nothing happens. 

**EXPECTED RESULTS:**

There should be no hidden content reachable using screen reader.
Note: If the hidden/invisible link is made visible on the page, then it should be made functional for screen reader user as well.

**ATTACHMENT(S):**

[ampstart_HiddenLink.mp4.zip](https://github.com/ampproject/ampstart/files/911801/ampstart_HiddenLink.mp4.zip)
